legal-api/report-templates/template-parts/bc-annual-report/legalObligations.html

```diff
@@ -0,0 +1,28 @@
+<div class="no-page-break">
+  <div class="section-title mt-4">Legal Obligations</div>
+
+  <div class="section-data mt-4">
+     You are required by the Business Corporations Act to keep the information about your corporation up
+      to date with the Registrar: For example, you must file annual reports, director changes and address changes.
+      The most common updates you are legally responsible to file include:
+  </div>
+  <div class="section-data mt-5">
+     <b>Annual Reports</b> - file an annual report each year within two months after
+      each anniversary of the date on which the company was recognized. This ensures your company remains in
+      good standing with the Registrar.
+  </div>
+  <div class="section-data mt-5">
+      <b>Director Changes</b> - update director information within 15 days of any
+      change (appointing, ceasing or updating an existing director's name or address)
+  </div>
+  <div class="section-data mt-5">
+     <b>Company Address Changes</b> - update any changes to a company's registered or records office addresses as
+     they occur.
+  </div>
+  <div class="section-data mt-5">
+     <b>Benefit Report</b> - The directors must produce and publish a benefit report on or before each annual
```

Comment:
1. The other items are plural. This one should be plural too for consistency.
2. "The" should not be capitalized, consistent with the other items.

Reply:
Fixed.